.gitignore

```diff
@@ -159,7 +159,7 @@ cmake-build-*/
 /test/compflags/sungeun/configs/type_variables/configTypeFun.good
 
 /test/chpldoc/compflags/combinations/*.doc.good
-
+!/test/chpldoc/compflags/combinations/license.doc.good
```

Comment:
What does the `!` here do?

Reply:
it excludes the file from being ignored. https://git-scm.com/docs/gitignore#_pattern_format